app/services/courses_query.rb

```diff
@@ -47,6 +49,22 @@ def visa_sponsorship_scope
       )
   end
 
+  def applications_open_scope
+    return @scope if params[:applications_open].blank?
+
+    @applied_scopes[:applications_open] = params[:applications_open]
+
+    @scope.where(application_status: Course.application_statuses[:open])
```

Comment:
Like I mentioned on the PR description, I am maintaining the same functionality as the live results page and if we decide to change this change would be easier by just changing this line.